packages/@react-native-windows/cli/src/index.ts

```diff
@@ -6,6 +6,19 @@
 
 import * as fs from 'fs';
 import * as path from 'path';
+import * as findUp from 'find-up';
+
+import * as appInsights from 'applicationinsights';
+
+appInsights.setup('795006ca-cf54-40ee-8bc6-03deb91401c3');
+const telClient = appInsights.defaultClient;
+if (!telClient.commonProperties.sessionId) {
+  telClient.commonProperties['sessionId'] = (new Date()).getTime().toString(36) + Math.random().toString(36).slice(2);
```

Comment:
```suggestion
  telClient.commonProperties['sessionId'] = crypto.randomBytes(32).toString('hex');
```

or similar.

Reply:
@NickGerleman I mentioned it on the PR description that a bunch of the content in this PR builds upon the previous one for project init, can you take a look at that one? also CC @dannyvv 
#6103